doc/en/components/layouts/divider.md

```diff
@@ -0,0 +1,190 @@
+
+## {Platform} Divider Example
+
+`sample="/layouts/divider/base", height="220", alt="{Platform} Divider Example"`
+
+<div class="divider--half"></div>
+
+## Dependencies
+
+<!-- WebComponents -->
+First, you need to install the {ProductName} npm package by running the following command:
+
+```cmd
+npm install {PackageWebComponents}
+```
+<!-- end: WebComponents -->
+
+<!-- React -->
+
+First, you need to the install the corresponding {ProductName} npm package by running the following command:
+
+```cmd
+npm install igniteui-react
+```
+
+You will then need to import the `Divider`, its necessary CSS, and register its module, like so:
+
+```tsx
+import { IgrDividerModule, IgrDivider } from 'igniteui-react';
+import 'igniteui-webcomponents/themes/light/bootstrap.css';
+
+IgrDividerModule.register();
+```
+
+<!-- end: React -->
+
+Before using the `Divider`, you need to register it as follows:
+
+
+```razor
+// in Program.cs file
+
+builder.Services.AddIgniteUIBlazor(typeof(IgbDividerModule));
+```
+<!-- Blazor -->
+
+You will also need to link an additional CSS file to apply the styling to the `Avatar` component. The following needs to be placed in the **wwwroot/index.html** file in a **Blazor Web Assembly** project or the **Pages/_Host.cshtml** file in a **Blazor Server** project:
+
+```razor
+<link href="_content/IgniteUI.Blazor/themes/light/bootstrap.css" rel="stylesheet" />
+```
+
+<!-- end: Blazor -->
+
+```ts
+import { defineComponents, IgcDividerComponent } from 'igniteui-webcomponents';
+
+defineComponents(IgcDividerComponent);
+```
+
+For a complete introduction to the {ProductName}, read the [*Getting Started*](../general-getting-started.md) topic.
+
+The `Divider` is capable of displaying images, initials, or any other content, including icons. Declaring an `Divider` is as simple as:
+
+```html
+<igc-divider></igc-divider>
+```
+
+```tsx
+<IgrDivider></IgrDivider>
+```
+
+```razor
+<IgbDivider></IgbDivider>
+```
+## Usage
+### Vertical Divider
+
+If the `vertical` attribute is set you the direction of the divider would be changed from horizontal to vertical.
+
+```html
+<igc-divider vertical></igc-divider>
+```
+
+```tsx
+<IgrDivider vertical></IgrDivider>
+```
+
+```razor
+<igrDivider Vertical></igrDivider>
+```
+
+`sample="/layouts/divider/vertical", height="330", alt="{Platform} Divider Vertical Example"`
+
+### Type
+
+The `type` attribute determines whether to render a `solid` or a `dashed` divider line. The default value is `solid`.
+
+```html
+<igc-divider type="dashed"></igc-divider>
+```
+
+```tsx
+<IgrDivider type="dashed"></IgrDivider>
+```
+
+```razor
+<IgbDivider Type="dashed"></igrDivider>
+```
+
+`sample="/layouts/divider/dashed", height="220", alt="{Platform} Divider Dashed Example"`
+
+### Middle
+
+```html
+<igc-divider middle="true"></igc-divider>
+```
+
+```tsx
+<IgrDivider middle="true"></IgrDivider>
+```
+
+```razor
+<IgbDivider Middle="True"></igrDivider>
```

Comment:
This section can be renamed to "Inset Divider" and elaborates more on possible options to have the divider inset from one or both sides. In general, ["Inset Divider" paragraph in Angular topic](https://www.infragistics.com/products/ignite-ui-angular/angular/components/divider#inset-divider) is a good guideline.
Additionally, the sample could be changed a bit to illustrate different views of the divider. Againg, Angular topic is a good example.